### DISCUSSION TITLE
Should the $SFL token be listed on Centralized Exchanges (CEXs)?

### DISCUSSION THREAD

**Discussion by poteitouweb3** [sunflower-land/sunflower-land#3539] [Polls]
### Listing $SFL on Centralized Exchanges may bring **positive** aspects:

**Increased Liquidity:** Centralized exchanges typically have a large user base, which can significantly increase the liquidity of the token. This makes it easier for users to buy and sell the token, fostering a more active and healthy market.

**Enhanced Visibility and Credibility:** Being listed on a reputable centralized exchange can enhance the visibility of the token to a broader audience. This can attract new players and investors to the game, which is crucial for the growth and sustainability of the GameFi ecosystem. Moreover, the process of getting listed often involves a due diligence check by the exchange, which can add a layer of credibility and legitimacy to the game and its token.

**Better Accessibility:** Centralized exchanges often provide user-friendly platforms that are accessible to a wide range of users, including those who are new to cryptocurrencies. This can make it easier for a broader audience to access the token, potentially increasing the game's player base.

**Market Stability:** Centralized exchanges often have measures in place to prevent market manipulation and to provide a more stable trading environment. This can be beneficial for the token's price stability, which is important for both players and investors.

**Regulatory Compliance:** Many centralized exchanges adhere to regulatory standards, which can be advantageous for the token in terms of compliance with financial laws and regulations. This is especially important in jurisdictions where regulations around cryptocurrencies are strict.

**Fiat Gateway:** Centralized exchanges often offer fiat-to-crypto trading pairs, allowing players and investors to buy the token directly with traditional currencies. This can significantly lower the entry barrier for new players who are not yet familiar with crypto.

**Marketing and Promotion:** Being listed on a centralized exchange often comes with marketing and promotional support, which can increase awareness and adoption of the GameFi project. This can include features on the exchange’s website, newsletters, and other marketing channels.

**Professional Support and Services:** Centralized exchanges provide professional support and services, including customer support, which can be beneficial for users who have queries or face issues with trading the token.

**Network Effects:** As more people trade the token on a centralized exchange, it can create a network effect, attracting even more users and contributing to the overall growth and success of the GameFi project.

**Partnership Opportunities:** Listing on a well-known centralized exchange can open up opportunities for partnerships and collaborations with other projects and companies in the crypto space, further enhancing the game’s ecosystem and potential for growth.

### Otherwise the **negative** aspects can be:

**Volatility Risk:** Centralized exchanges often have a wider investor base, including speculative traders, which can lead to increased price volatility. Speculators may not be interested in the game itself but rather in profiting from short-term price movements, potentially leading to large price swings.

**Market Manipulation:** While centralized exchanges take measures to prevent manipulation, they are not immune to it. Large holders or “whales” can influence the market, causing sudden price changes that may not reflect the token's true value or the health of the underlying GameFi project.

**Dependence on Exchange Stability:** The token's value and liquidity become partly dependent on the stability and reliability of the exchange. Any technical issues, security breaches, or regulatory problems faced by the exchange can negatively impact the token.

**Impact on Tokenomics:** The economics of the token within the game might be affected by its trading on external markets. This could lead to imbalances in the in-game economy, especially if the token's primary use case is within the game itself.

Based on both sides and any other concern you may have, please vote!

**Comment by rodolfoddias** [comment]
Aren't we already listed in CoinEx?

I guess a better move could be advertise it instead of listing in other CEX. It's expensive to do it and they did it for free.

**Comment by adamhannigan** [comment]
@poteitouweb3 Great discussion! This isn't something we often talk about publicly as we don't want any false hope/promises. I'll put some notes down regarding central exchanges.

The main purposes of listing SFL on exchanges would be:

1. Make is easier for gamers to attain SFL to use in-game. Currently players use DEXes which are a nightmare - especially for new players
2. Increase credibility + reputation of project to attract more players

We would not list on an exchange to pump bags temporarily. There is no team SFL allocation, so we are not interested in these short term gains either. The choice would definitely be factored on longevity and utility of the token: this would eliminate a few exchanges known for ponzinomics and pump/dumps.

Our legal team is always keeping up to date with industry standards and is currently in the process of giving advice around listing the token publicly. We have already satisfied the conditions that the SFL token is not a financial product - our game is extremely conservative compared to other games. We do not offer staking, interest, governance or other investment vehicles.

This structure we have around the token definitely makes it easier to list.

When it comes to publicly listing, there are a few hurdles:

1. Will people buy the token on an exchange to use it in-game OR will they buy it to hold for expected profits
2. Almost every big exchange requires you provide a market maker. This is a company you give 500K+ USD liquidity to control the price. Unfortunately what we are seeing a lot from these MMs is market manipulation to hype up token prices & set 'false floors' - they are specifically looking for short term speculation to earn their % and no interest in longevity.

At the end of the day, the decision around a public listing will lean on:

1. Regulations
2. Ecosystem is ready - economics are sound and a public listing will support longevity.

I hope these notes help ❤️

**Reply by poteitouweb3** [reply]
Awesome Adam! Nothing different from what i expect from you guys: Caring about the game that is bringing a lot of quality along these 2+ years! I share the same reasons as you and unfortunately you're completely right about many MMs not caring about the project.

Hope someday this is going to make sense for the game and hope there will be a truly place to arrive 💯

**Reply by NotKeen** [reply]
Maybe we could start by re-evaluating / rebuilding our QuickSwap / Wishing Well system. There has been some discussion about this in the forum and some interesting points by Leidenfrost:
https://discord.com/channels/880987707214544966/1217387658943135744

My takeaways:

- WW contract might be broken? (more WW tokens existing than pool tokens)
- Wishing well doesn't attract professional liquidity providers, maybe pivot from giving LP incentives in-game to externally (maybe via the QuickSwap rewards mechanics)
- AFAIK our SFL-MATIC pool is on QuickSwap v2, maybe consider a switch to v3?

**Reply by adamhannigan** [reply]
We will look into these 👍  Cheers!

There are some restrictions to what sort of incentives can be offered for LP. We can't support any staking mechanisms as these often reflect financial products.

My understanding is many of the external reward programs require the player to stake their LP.

Any ideas around this?

**Reply by rodolfoddias** [reply]
Hey @adamhannigan are there any restrictions to reward holders based on how many tokens they have in their wallet? This way we wouldn't relay on staking mechanisms. We could do it based on weekly snapshots for, I don't know.. community coins and other NFTs / Block Bucks? We could do something similar to the BUD's chest, where holders have the chance to open "x" treasure chests if they hold "X" tokens. More tokens = More days with available chests, just like the new bud chest feature. 

I guess we could do the same with Quickswap LP tokens and SFL tokens. Also reward things that can't be traded, so it wouldn't be considered a financial incentive. What are your thoughts regarding this approach?

It would be great if the players were able to provide liquidity trough the game itself and don't need to withdraw tokens in order to do it - due the fees. I guess this would be a HUGE incentive for players to actually provide liquidity - specially if they get a chance to earn Block Bucks. It would also make it easier for web 2 players to buy the token, since they wouldn't need to leave the game - But I guess it would mean we would either have to integrate with QuickSwap or develop our own DEX (I do know someone that could build it, by the way).

I would also consider a cap, to prevent the financial product sentiment. If you have this, you earn one treasure chest per day (or key) per day. Considering keys as rewards, we could introduce a new Goblin like the Head Goblin of Harry Potter. The ones who have the keys to the vaults:

![image](https://github.com/sunflower-land/sunflower-land/assets/11824506/76a1ed6c-c367-458d-b956-539624f1e50f)

**Reply by 0xLeidenfrost** [reply]
Aye, distributing incentives through a mechanism that requires bonding/staking typically raises the financial product and/or derivatives classification. A lot of protocols, fintech apps, and games like to falsely use that as a mechanism for controlling price ( e.g. reduce sell pressure from an initial distribution), whereas one of the only real use cases for staking is to ensure the consensus continuity of an L1 chain - something none of those apps, protocols, or games actually are.

From what I've seen, avoiding this, while also steering clear of a fintech classification, is usually done by:
* Having the distribution of incentives under the control of the community (governance) 
* Having the incentives come out of a shared community treasury funded by mandatory protocol-level functionality (TX fees, burns, contract invocation fees, etc.

Please read my discord comments, which include some more aids (from Quickswap analytics), but to sum it up:
1. The design of the wishing well system is hurting players and alienating investors.
2. The risk profile of $sfl/$matic LPs is on average twice higher than that of an ethereum/matic consensus staker because they're exposed to two assets at the same time.
3. Indefinitely locking player funds in the $sfl/$matic pool - so they qualify for the in-game distribution of $sfl - is crippling their ability to react to major market moves (and say move to the sfl/usdc pool for the duration of the move). The Quickswap analytics impermanent loss page speaks plenty on this.
4. Awarding pool incentives in-game signals to external investors there's no room for them in SFL LP pools.
 -Unless they're actively playing SFL, there's no need for them to deal with our liqudity, as they'll never receive incentives.
 -No one serious about LP-ing funds would be willing to work with a pool that just has a transaction fee rewards structure and makes them permanently positioned in it.

Now, since on-chain governance is something I've not seen discussed, but I do know there's a community treasury, I do feel this whole topic can receive more consideration.

I personally think, there's more awkwardness around the wishing well structure, like the wishing well token used for representing LP share (which is already represented by the LP pool token LPs get from the quickswap contract), or WW token supply mismatch issue Elias described in their post. But in my mind, deepening liquidity and making the whole process a lot more user-friendly and calibrated to the risk-profile they're supposed to take on is more of a pressing issue. Specifically:
* Reduce the impermanent loss heat all LPs are feeling, which for the most part cannot be compensated with the current rewawrd structure
* Find a way not to alienate external investors by 1) having the incentive stucture depend on participation in the game, 2)  removing their abilityh to use what little counter-measure defi has to offer for protecting their funds like moving to the sfl/matic pool for a short period of time (e.g. 24hrs) during major market moves.

$SFL/$MATIC annual impermanent loss: https://quickswap.exchange/#/analytics/v2/pair/0x6f9e92dd4734c168a734b873dc3db77e39552eb6
$SFL/$USDC annual impermanent loss: https://quickswap.exchange/#/analytics/v2/pair/0x6f09e1a2e6fb877609396082f21aaba7819fec71

**Comment by eliasSFL** [comment]
> Maybe we could start by re-evaluating / rebuilding our QuickSwap / Wishing Well system. There has been some discussion about this in the forum and some interesting points by Leidenfrost: https://discord.com/channels/880987707214544966/1217387658943135744
> 
> My takeaways:
> 
> * WW contract might be broken? (more WW tokens existing than pool tokens)
> * Wishing well doesn't attract professional liquidity providers, maybe pivot from giving LP incentives in-game to externally (maybe via the QuickSwap rewards mechanics)
> * AFAIK our SFL-MATIC pool is on QuickSwap v2, maybe consider a switch to v3?

Here's the discussion on github #3512

**Comment by WWW-Spaceships** [comment]
Adam thanks for sharing what you do about the financial product piece that helps us to understand why DEFI is more incorporated into the game. I personally wish we could provide liquidity but did not have to remember to wishing well. That leaves so much to chance and memory.

**Comment by adamhannigan** [comment]
Just to circle back, our lawyers have given us more guidance around listing on public exchanges.

Given SFL's current legal status, there is no issues listing since the function of the token is clear.

Where it gets a bit grey is if majority of holders and transactions reside on the public exchanges. This could change the image of it being a utility token - for instance, 90% people just hold it on exchanges in hopes value goes up. Overall it is positive news on what we can do 👌
